web/package.json

```diff
@@ -86,12 +89,12 @@
     "react-icons": "^4.7.0",
     "react-map-gl": "^7.0.19",
     "react-router-dom": "6.14.2",
-    "react-spinners": "^0.13.6",
-    "react-spring-bottom-sheet": "3.5.0-alpha.0",
-    "recharts": "^2.1.16",
+    "react-spinners": "^0.13.8",
+    "react-spring-bottom-sheet": "3.4.1",
+    "recharts": "^2.7.3",
```

Comment:
There's quite a few bumps of minor version, did you test that they are not breaking anything? (you can't always trust people use SemVer correctly 😉)

Reply:
For recharts in particular we only use it for the gauges so their changes don't really affect us at all.